src/components/Item/VisualizationItem/Visualization/IframePlugin.js

```diff
@@ -146,14 +146,15 @@ const IframePlugin = ({
         return error === 'missing-plugin' ? (
             <div style={style}>
                 <MissingPluginMessage
-                    //item={item}
+                    itemType={itemType}
                     dashboardMode={dashboardMode}
                 />
             </div>
         ) : (
             <div style={style}>
                 <VisualizationErrorMessage
-                    //item={item}
+                    itemType={itemType}
+                    visualizationId={visualization.id}
```

Comment:
Is this the same as what was being calculated with `getVisualizationId` before?

Reply:
Yes:
https://github.com/dhis2/dashboard-app/pull/2199/files#diff-009a33196703f2c1b54a4360314f7e91351dc9b2323826f8a776c13d98ddc7a5L30
and
https://github.com/dhis2/dashboard-app/blob/master/src/modules/item.js#L21

Which is the AO in the `item` object, so the `id` is the AO's `id'.